test/manual/src/manualTests.ts

```diff
@@ -1,4 +1,4 @@
-import { BankIdClient } from "bankid";
+import { BankIdClient } from "../../../";
```

Comment:
Had to make this change for the test to run. Possibly my local env is misconfigured, or the package.json root config is?

Reply:
These tests are suppose to run with a "linked" local BankID build, so let's revert.

```suggestion
import { BankIdClient } from "bankid";
```